app/views/donations/_donation_form.html.erb

```diff
@@ -39,7 +39,7 @@
       <div class="col-xs-8 col-md-6 col-lg-3">
         <%= f.association :product_drive_participant,
                           collection: @product_drive_participants,
-                          selected: donation_form.product_drive_participant_id,
+                          selected: new_purchase_default_location(@donation),
```

Comment:
Can we rename this method so it doesn't have `purchase` in its name? You can move it to ApplicationHelper.

Reply:
Resolved in [884500a](https://github.com/rubyforgood/human-essentials/pull/4722/commits/884500aa161f83b69b6116857be567a9728c678f)